src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  margin: 0;
```

Comment:
Don't reset your margins in * selector, you should reset it for specific tags in reset.css